### PR TITLE
Elasticbeanstalk env tags

### PR DESCRIPTION
**Pull request by sidewinder12s**
Integrate Elasticbeanstalk env tags branch & option_settings fix.

### CODE REVIEW

**Comment by sidewinder12s** [comment]
@rohitkothari Hey can you check out this pull request? I updated it with a fix for a bug & document updates beyond what atward had done.

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ Ansible modules for working with Amazon Elastic Beanstalk
 Requirements
 ------------
 
-This module requires [boto](https://github.com/boto/boto)
+elasticbeanstalk_app & elasticbeanstalk_version require [boto](https://github.com/boto/boto)
```
Comment (rohitkothari) [inline README.md]: Does this mean that boto should also be imported along with boto3 in python code?

Comment (sidewinder12s) [inline README.md]: Nope since they are 3 separate modules. Just that you need both boto & boto3 to run all three modules.

Comment (rohitkothari) [inline README.md]: Oh right, I didn't realize this as the other two modules were not present in the pull request, and this was just readme change.

File: library/elasticbeanstalk_env.py [hunk 2]
```diff
@@ -232,6 +239,8 @@ def check_env(ebs, app_name, env_name, module):
 
     module.exit_json(**result)
 
+def filter_empty(**kwargs):
```
Comment (sidewinder12s) [inline library/elasticbeanstalk_env.py]: Ohhh, I think this is removing fields that are not defined from the aws request.

**Comment by rohitkothari** [comment]
Since you have tested it's working -
:+1:

**Comment by sidewinder12s** [comment]
👍

**Pull request merged**